packages/menu/src/styles.tsx

```diff
@@ -135,6 +136,7 @@ export const activeMenuItemContainerStyle = css`
 export const activeTitleTextStyle = css`
   font-weight: bold;
   color: ${palette.white};
+  text-shadow: unset !important; // no pseudo bold text when active
```

Comment:
@TheSonOfThomp , would this work? Maybe instead of `important` you can take advantage of the CSS variable fallback, so for example:

```
text-shadow: var(--text-shadow, 0.125px 0 var(--lg-menu-item-text-color),
      -0.125px 0 var(--lg-menu-item-text-color),
      0 0.125px var(--lg-menu-item-text-color),
      0 -0.125px var(--lg-menu-item-text-color));
```

then you can do this instead:
```
export const activeTitleTextStyle = css`
  font-weight: bold;
  color: ${palette.white};
  --text-shadow: unset;
`;
```

Reply:
it doesn't seem to work with `unset` but it works with `none`

Reply:
Unfortunately no (at least not like this), since the issue is the hover styles have a higher specificity than the text styles. I'll take a deeper look at getting rid of the `!important`

Reply:
Looks like I have it working with a slightly different `var` approach